packages/oidc-middleware/src/oidcUtil.js

```diff
@@ -44,7 +44,7 @@ oidcUtil.createClient = context => {
     issuer,
     client_id,
     client_secret,
-    redirect_uri,
+    loginRedirectUri: redirect_uri,
```

Comment:
Just to double check, here we need `loginRedirectUri` but not `logoutRedirectUri`?